packages/grafana-ui/src/components/BigValue/BigValue.story.tsx

```diff
@@ -32,11 +14,84 @@ export default {
     docs: {
       page: mdx,
     },
+    knobs: {
+      disabled: true,
+    },
+  },
+  argTypes: {
+    width: { control: { type: 'range', min: 200, max: 800 } },
+    height: { control: { type: 'range', min: 200, max: 800 } },
+    colorMode: { control: { type: 'select', options: [BigValueColorMode.Value, BigValueColorMode.Background] } },
+    graphMode: { control: { type: 'select', options: [BigValueGraphMode.Area, BigValueGraphMode.None] } },
+    justifyMode: { control: { type: 'select', options: [BigValueJustifyMode.Auto, BigValueJustifyMode.Center] } },
+    textMode: {
+      control: {
+        type: 'select',
+        options: [BigValueTextMode.Auto, BigValueTextMode.Name, BigValueTextMode.ValueAndName, BigValueTextMode.None],
+      },
+    },
+    color: { control: 'color' },
+    sparkline: {
+      table: {
+        disable: true,
+      },
+    },
+    onClick: {
+      table: {
+        disable: true,
+      },
+    },
+    className: {
+      table: {
+        disable: true,
+      },
+    },
+    alignmentFactors: {
+      table: {
+        disable: true,
+      },
+    },
+    text: {
+      table: {
+        disable: true,
+      },
+    },
+    count: {
+      table: {
+        disable: true,
+      },
+    },
+    theme: {
+      table: {
+        disable: true,
+      },
+    },
```

Comment:
not sure I understand why we need these

Reply:
Maybe so they don't show up as fields as these are not primitives? Would also like to know

Reply:
Yeah...you're right Hugo